test/stress/classic/README.md

```diff
@@ -0,0 +1,27 @@
+# Stress Test - Classic
```

Comment:
You can set up a local BanyanDB with OAP by following the instructions provided in this document..